images/macos/provision/core/openssl.sh

```diff
@@ -13,21 +10,23 @@ if is_BigSur; then
     exit 0
 fi
 
-# Install OpenSSL 1.0.2t
-# https://www.openssl.org/policies/releasestrat.html - Version 1.0.2 will be supported until 2019-12-31 (LTS)
-# To preserve backward compatibility with ruby-toolcache
-/usr/local/bin/brew tap-new local/openssl
-FORMULA_PATH=$(/usr/local/bin/brew extract openssl local/openssl | grep "Homebrew/Library/Taps")
-/usr/local/bin/brew install $FORMULA_PATH
+if is_Catalina; then
```

Comment:
May be
```
if ! is_Catalina; then 
  exit 0;
fi

```
???

Reply:
Let's remove OpenSSL for all platforms including Catalina

Reply:
agreed
changed to this approach

Reply:
The condition has been removed. We'll not install SSL 1.0.2 in all macOS.